R/plot.R

```diff
@@ -573,27 +498,66 @@ plot_predictions <- function(data = NULL,
       )
   }
 
-  plot <- plot +
-    ylab("True and predicted values")
-
-  # facet if specified by the user
-  if (!is.null(facet_formula)) {
-    if (facet_wrap_or_grid == "facet_wrap") {
-      plot <- plot +
-        facet_wrap(facet_formula, scales = scales, ncol = ncol)
-    } else {
-      plot <- plot +
-        facet_grid(facet_formula, scales = scales)
-    }
-  }
-
   return(plot)
 }
 
 
 
+#' @title Filter Data Used for Plotting
+#'
+#' @description
+#' Filter data before it gets passed to [plot_predictions()] using arbitrary
+#' filter statements. In contrast to the usual behaviour of filtering functions,
+#' rows do net get removed, but instead corresponding values of either
```

Comment:
I understand the motivation but I don't know if it's the best idea to call it `filter()` then.

Reply:
it could be called `keep()` or `keep_rows()` or `keep_data()`? Or alternatively `remove_rows()` or `remove_data()` or `remove()` and we get rid of the negation internally.

Reply:
Agree with @Bisaloo. Another option would be filter_for perhaps but still close to filter nomenclature.

Reply:
Do you have a preference?

Reply:
Maybe literally calling it `make_na()` is good